Assets/Scripts/Layers/LayerUI.cs

```diff
@@ -198,7 +198,7 @@ private void OnLayerChildrenChanged()
             RecalculateCurrentTreeStates();
         }
         
-        private void OnParentOrSiblingIndexChanged(int newSiblingIndex)
+        private void OnParentOrSiblingIndexChanged(int newSiblingIndex, LayerData data)
```

Comment:
Deze change is niet meer nodig toch? Ik zie niet dat `LayerData data` gebruikt wordt. Zo ja, graag even terugdraaien om te voorkomen dat we changes krijgen die geen functie hebben in de applicatie

Reply:
Laat maar, ik heb gemist dat die onderin wel gebruikt wordt

Reply:
Ik vraag me wel af of dit niet dubbelop is, want de SiblingIndex is toch te krijgen uit LayerData? Zou het niet beter zijn om dan de hele SiblingIndex eruit te vervangen?

Reply:
Dit was ivm het layermoved event, ik ging ervanuit als je in de toekomst op dit event wil subscriben dat je dan wel echt wil weten welke layerdata er gemoved is

Reply:
Door op dit event te subscriben heb je al een referentie naar de layer, dan is dit niet nodig toch?

Reply:
is nu aangepast